videoframegrid.py

```diff
@@ -68,6 +71,28 @@ def swap_frame(self, frame):
             else:
                 self.selected_frame = frame
 
+    def toggle_fullscreen(self, selected_frame):
+        if not self.fullscreen:
+            for videoframe in self.videoframes:
```

Comment:
I don't really find:
```
[videoframe.hide()
 videoframe.player.audio_set_mute(True)
 for videoframe in self.videoframes
 if videoframe != selected_frame]
```
that much more clearer to read, but it could be that I'm not used to it. Perhaps you got some other ninja solution though? :)